airflow/providers/pgvector/provider.yaml

```diff
@@ -43,7 +43,9 @@ integrations:
 dependencies:
   - apache-airflow>=2.7.0
   - apache-airflow-providers-postgres>=5.7.1
-  - pgvector>=0.2.3
+  # limiting to <=0.2.5 version due to https://github.com/pgvector/pgvector-python/issues/79
+  # observed in 0.3.0.
+  - pgvector>=0.2.3,<=0.2.5
```

Comment:
Better to set `!=0.3.0` . Seems that the fix is already merged and scheduled to be released in 0.3.1, so `!=0.3.0` is also future-proof